detect-hidden-unicode/action.yml

```diff
@@ -0,0 +1,80 @@
+name: "Detect hidden unicode"
+description: "Detects hidden unicode in newly committed config files."
+
+inputs:
+  python-version:
+    description: "Python version to use for running the action"
+    default: "3.10"
+  poetry-version:
+    description: "Use a specific poetry version. By default the latest release is used."
+  cache-poetry-installation:
+    description: "Cache poetry and its dependencies. Default is 'true'. Set to an other string then 'true' to disable the cache."
+    default: "true"
+  github-token:
+    description: "GH Token for writing/editing a comment to/in a PR"
+    required: true
+  pr-comment:
```

Comment:
Is that a requirement somehow that this is optional?
I'd assume when developers include this action they also want the information and dont see how it hurts to have it...
not much but saves a few lines that dont need to be maintained then